centaur/src/main/resources/standardTestCases/drs_usa_jdr_preresolve.test

```diff
@@ -25,7 +29,7 @@ metadata {
   "outputs.drs_usa_jdr.size2" = 38.0
   # This JDR file has a gsUri that doesn't end in /fileName so it must be downloaded with the DRS localizer
   "outputs.drs_usa_jdr.cloud1" =
-    "/cromwell_root/jade.datarepo-dev.broadinstitute.org/v1_f90f5d7f-c507-4e56-abfc-b965a66023fb_585f3f19-985f-43b0-ab6a-79fa4c8310fc/hello_jade.json"
+    "/cromwell_root/CromwellSimpleWithFilerefs/hello_jade.json"
```

Comment:
Is this safe enough? When we choose the localization path, it seems to start with the data repo hostname which is not under user control. However, the preferred localization path could be anything, and I don't know what else might be in `/cromwell_root/` that a "creative" localization path could overwrite and lead to undesirable consequences.

Reply:
This is a good point... maybe we want to stick these all in a dedicated subdir? `/cromwell_root/localized_inputs/...`

Reply:
I can add a directory here. This would currently only come into play for DRS inputs with a specified localization path so I'll try to come up with something that reflects that and see what my reviewers think. 🙂

Reply:
@mcovarr, out of curiosity, was I right to worry about this? Are there other files that exist in `/cromwell_root/` that could accidentally be overwritten by a carefully-crafted localization path?

Reply:
Cromwell does localize its various scripts here and if e.g. TDR told us to write a file named `script.sh` that would certainly cause an issue for the job execution. 🙂